rzk/src/Rzk/Main.hs

```diff
@@ -58,14 +59,16 @@ parseRzkFilesOrStdin = \case
     rzkModule <- parseStdin
     return [("<stdin>", rzkModule)]
   -- otherwise — parse all given files in given order
-  paths -> forM paths $ \path -> do
-    putStrLn ("Loading file " <> path)
-    result <- Rzk.parseModule <$> readFile path
-    case result of
-      Left err -> do
-        putStrLn ("An error occurred when parsing file " <> path)
-        error err
-      Right rzkModule -> return (path, rzkModule)
+  paths -> do
+    expandedPaths <- foldMap glob paths
```

Comment:
For each path in `paths`, you should throw an error, if it corresponds to an empty list! Right now it silently says "Everything is ok!" if the glob does not refer to any files (or if there is a typo in a filename).

Reply:
I will fix that.